### PR TITLE
Check that the user's account balance is less than required and raise…

### PR DESCRIPTION
**Pull request by vasiliy-zaznobin**
… an RuntimeError

### CODE REVIEW

File: proxy/testing/testing_helpers.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def web3(self) -> Web3:
 
 
 def request_airdrop(address):
-    FAUCET_URL = os.environ.get('FAUCET_URL', 'http://faucet:3333')
+    FAUCET_URL = os.environ.get('FAUCET_URL', 'http://localhost:3333')
```
Comment (afalaleev) [inline proxy/testing/testing_helpers.py]: Why did you change this URL ?
@vakond resolved this issue.

Comment (vasiliy-zaznobin) [reply proxy/testing/testing_helpers.py]: Now the localhost is used for the local config. See https://github.com/neonlabsorg/proxy-model.py/blob/3085e0b17cd3c14bda8d97d98030feca921b26c3/proxy/docker-compose-test.yml#L110-L111

Comment (afalaleev) [reply proxy/testing/testing_helpers.py]: This line means, that container binds the container port 3333 to the host ip:port 127.0.0.1.
But test runs in test-container, not on the host.

Comment (vasiliy-zaznobin) [reply proxy/testing/testing_helpers.py]: Fixed

**Review by afalaleev — APPROVED** [review]
